admin/app/components/solidus_admin/ui/forms/checkbox/component.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="flex gap-2 items-center">
+  <label class="flex gap-2 items-center">
+    <%= hidden_field_tag @name, "0" %>
+    <%= render component("ui/checkbox").new(
+      name: @name,
+      value: "1",
+      checked: @checked,
+      **@attributes
+    ) %>
+    <%= caption %>
```

Comment:
Caption seems like a weirdly named attribute for the label. Can we name this duck as it quacks please? ;)